tests-invoke

```diff
@@ -77,6 +77,9 @@ def main():
             entry_point = alt_entry_point
         cmd = ["timeout", "120m", entry_point]
 
+        os.environ["TEST_REVISION"] = opts.revision
```

Comment:
We already [do this](https://github.com/cockpit-project/cockpit/blob/main/test/common/run-tests#L528)

Reply:
But that's wrong half of the time, no? `HEAD` is the result of the rebase. We could use `ORIG_HEAD`, but that then assumes that a rebase has just been done, which might not be true.

I think it is best to set the values here, with authority.

Reply:
Ah, and you need the original HEAD from the proposed branch, without a rebase? AFAICS the only place where we use this variable so far is in bots/test-failure-policy, and there it doesn't matter much. OK then.

Reply:
I'd say the only reason to have this in the environment at all is to reliably point to the pre-rebase HEAD.  If you want HEAD, you can just look at it with `git rev-parse HEAD`.